src/containers/CustomTable/index.tsx

```diff
@@ -126,15 +121,15 @@ class CustomTable extends React.Component<Props, State> {
                 );
               })}
               {emptyRows > 0 && (
-                <TableRow style={{ height: 48 * emptyRows }}>
+                <TableRow style={{ height: 48 / emptyRows }}>
```

Comment:
Why is it necessary?

Reply:
It removes the whitespace below the results in the table if there's less than 50 results. An example query to try with it is: Level Equals 1 on Blocks

Reply:
Please remove the `{emptyRows >0 && }`

Reply:
Removed and pushed.